client/my-sites/media-library/content.jsx

```diff
@@ -136,11 +144,19 @@ module.exports = React.createClass( {
 					);
 					break;
 				default:
-					message = this.translate(
```

Comment:
While it perpetuates the Flux implementation of media, for consistency's sake, I think we should be implementing media exceeding allocated storage with the [other validation checks in `MediaValidationStore`](https://github.com/Automattic/wp-calypso/blob/c322206af0fd40f7d6ef1c81912f895c94606c2f/client/lib/media/validation-store.js#L53-L73).